fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -456,6 +456,7 @@ public void testHeadExternalDatastreamRedirect() throws IOException, ParseExcept
             assertEquals(TEMPORARY_REDIRECT.getStatusCode(), response.getStatusLine().getStatusCode());
             assertEquals("http://example.com/test", getLocation(response));
             assertEquals("bytes", response.getFirstHeader("Accept-Ranges").getValue());
+            assertEquals("0", response.getFirstHeader("Content-Length").getValue());
```

Comment:
It's too bad this comes through as a Content-Length = 0, which seems slightly more misleading than not specifying a content length, but I guess that's just the way it is in order to keep clients happy.

Reply:
I can take a better look when I have a chance, but for a `GET` request I am assuming that this is correct as the body length is 0. For a `HEAD` request I think it should be removed completely.

Reply:
I'm okay with this going in as is. If we are going to have Content-Length on other HEAD requests, then it makes sense to have a consistent response for redirect binaries across HEAD and GET. Was just lamenting that it could be confusing to see a content-length of 0 when that's not the actual length, but its just the first hop in a redirect so it should be fine.